roles/heat/templates/etc/heat/heat.conf

```diff
@@ -11,6 +11,8 @@ notification_driver = log
 log_dir = /var/log/heat
 use_stderr = false
 
+convergence_engine = false
```

Comment:
define variable in roles default and use that variable here in template. 
in roles/heat/defaults/main.yml
heat:
  convergence_engine = false

and than in  roles/heat/templates/etc/heat/heat.conf 
convergence_engine = {{ heat.convergence_engine }}